Dockerfile.npm

```diff
@@ -0,0 +1,7 @@
+FROM node:6.14.1 as npm
```

Comment:
Can't get `npm install` to work on Alpine/Slim.

Reply:
Given the size of `node_modules` (~500 MB) it's likely there are many library dependencies for compiling stuff or needed at runtime. This base image will be cached on the `containers` node.